tests/unit/components/status/HeatMapColorBar.spec.js

```diff
@@ -0,0 +1,227 @@
+    //                                                   which is not available for vue-test-utils as it only can understand virtualDOM elements and not native Javascript
+    //                                                   document or (DOM) objects so the failure happens when we assert the same.
+    //
+    //                                                   So the below lines are commented to be validate via VRT.
+    // expect(target_div_gradient_holder.attributes().style).toBe(
+    //   "height: 400px;"
+    // );
+  });
+  test("Given that the uuid is null, upper_range, lower_range and height, When mounted successfully, Then prop update of the heatmap_height to 400, assert height is updated to 400 and as uuid is empty then gradient_range values are applied on linear-gradient", async () => {
+    const propsData = {
+      upper_range: 100,
+      lower_range: 0,
+      gradient_uuid: "",
+      heatmap_height: 481,
+      units: "mA",
+      gradient_range: [
+        { color: "#2c7bb6", offset: "0%" },
+        { color: "#00a6ca", offset: "25%" },
+        { color: "#00ccbc", offset: "50%" },
+        { color: "#90eb9d", offset: "75%" },
+        { color: "#ffff8c", offset: "100%" },
+      ],
+    };
+
+    wrapper = mount(ComponentToTest, {
+      propsData,
+      localVue,
+    });
+
+    const target_div_gradient_holder = wrapper.find(
+      ".div__heatmap-gradient-holder"
+    );
+
+    expect(target_div_gradient_holder.attributes().style).toBe(
+      "height: 481px;"
+    );
+
+    // await wrapper.setProps({ heatmap_height: 400 });  Currently in order to match the feature of changing gradient and height native
+    // await wrapper.vm.$nextTick();                     Javascript functions are invoked which calls functions removeChild(object<--div__heatmap-gradient-holder),
+    //                                                   createElement('div') and appendChild(object<-- new div__heatmap-graident-holder)
+    //                                                   This results in the passed prop of heatmap_height been applied on new object of div__heat-gradient-holder)
+    //                                                   which is not available for vue-test-utils as it only can understand virtualDOM elements and not native Javascript
+    //                                                   document or (DOM) objects so the failure happens when we assert the same.
+    //
+    //                                                   So the below lines are commented to be validate via VRT.
+    // expect(target_div_gradient_holder.attributes().style).toBe(
+    //   "height: 400px;"
+    // );
+  });
+
+  // test("Given that the uuid is empty, upper_range, lower_range and height, When mounted successfully, Then update the uuid to valid value, and as uuid is match a default linear-gradient is applied on HeatMapColorBar", async () => {
```

Comment:
We discussed at the end of our call this morning deleting commented code before submitting the Pull Request for review. Please remove this before we merge

Reply:
removed